README.md

```diff
@@ -20,120 +36,43 @@ First, build the app locally.
+`yarn demo:pd` — starts the development servers for the Spotlight API and the Public Dashboard client in "demo mode" by supplying the necessary environment variables in the command line. (This will run your local site off of fake data fixtures rather than live data from a calculation pipeline; see the [Spotlight API documentation](spotlight-api/#demo-mode) for more information.)
 
-We use [`@testing-library/react`](https://testing-library.com/docs/react-testing-library/intro)
-for component tests you will want to import the testing-library functions via `src/testUtils.js`
-instead; it re-exports the full API from `@testing-library/react` with a wrapper around `render`
-that includes any globally expected React Context providers (e.g. the `ThemeProvider` for `styled-components`).
+### Other tools
 
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+Style and formatting rules for this repository are defined with [ESLint](https://eslint.org/) and [Prettier](https://prettier.io/). The base configuration lives in the root of this repository and is extended by the individual packages as necessary. For this reason, it is important to run the individual `lint` commands for each package rather than trying to lint the entire repository at once with `eslint .` — this will exclude the nested configurations and produce inconsistent results.
 
-### `yarn lint`
+Linting rules (including auto-fixing) are applied to changed files in a pre-commit hook using [Husky](https://github.com/typicode/husky) and [lint-staged](https://github.com/okonet/lint-staged). These are configured in the root `package.json`.
 
-Runs the eslint checks against the repository to check for issues in code style.
+We use [Github Actions](https://docs.github.com/en/free-pro-team@latest/actions) for continuous integration tasks, which include lint checks as well as automated JavaScript tests where applicable.
 
-Eslint rules are configurable in `.eslintrc.json`. Any change to this file should be accompanied with an explanation
-for the change and why it should be merged.
+## Adding new packages
 
-### `yarn eject`
+Packages themselves do not have any special requirements under Yarn Workspaces, nor are they required to use the same packages or versions as any other Workspace; just create a new directory, **add it to the `workspaces` list in the root `package.json`**, and then run `yarn init` in your package directory as you normally would. (The entire repo is covered by a GPL V3 license, so make sure your package's `license` field conforms to that.)
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+Do note that there should only be one `yarn.lock` file for the entire repository; if one is created in your new package directory, you should delete it, make sure the directory is properly listed in the `workspaces` list, and re-run `yarn install`.
```

Comment:
So although there are separate `package.json` files per package, there is only a single resulting `yarn.lock` file. And the possibility of conflicts between the two files gets smoothed out by Yarn Workspaces?

Reply:
yes ... if there are redundancies it "hoists" them to the root to avoid duplicate installs, and if there are multiple versions that's fine because that happens all the time in node anyway. We will get more mileage out of the hoisting once we have multiple React applications in here